src/boot/efi/vmm.c

```diff
@@ -83,6 +83,9 @@ EFI_STATUS vmm_open(EFI_HANDLE *ret_vmm_dev, EFI_FILE **ret_vmm_dir) {
         assert(ret_vmm_dev);
         assert(ret_vmm_dir);
 
+        /* Make sure all file systems have been initialized. */
```

Comment:
maybe add a comment here, why this is assumed to be safe, and why it isn't assumed to be safe outside of the VM context. Otherwise the next person will move this back into efi_main()...